tests/unit/metadata/test_single_table.py

```diff
@@ -105,6 +109,183 @@ def test__load_from_dict(self):
         assert instance._alternate_keys == []
         assert instance._constraints == []
 
+    @patch('sdv.metadata.single_table.Path')
+    def test_load_from_json_path_does_not_exist(self, mock_path):
+        """Test the ``load_from_json`` method.
+
+        Test that the method raises a ``ValueError`` when the specified path does not exist.
+
+        Mock:
+            - Mock the ``Path`` library in order to return ``False``, that the file does not exist.
+
+        Input:
+            - String representing a filepath.
+
+        Side Effects:
+            - A ``ValueError`` is raised pointing that the ``file`` does not exist.
+        """
+        # Setup
+        mock_path.return_value.exists.return_value = False
+        mock_path.return_value.name = 'filepath.json'
+
+        # Run
```

Comment:
small nit: I have seen the comment `# Run and Assert` in cases like these.

Reply:
Nice catch, I usually write `# Run / Assert` but this time I missed it. WIll address it asap.